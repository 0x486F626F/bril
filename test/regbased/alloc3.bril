main {
	vj: int = const 1;
	r_03: int = const 2;
	r_01: int = const 1;
	r_04: int = const 2;
	v3: int = const 3;
	v0: int = const 5;
	r_02: int = add vj r_01;
	v1: int = id r_01;
	r_01: int = sub r_03 v0;
	vf: int = id r_02;
	r_02: int = id vf;
	r_02: int = mul r_02 r_01;
	vh: int = id r_01;
	r_01: int = add vj r_04;
	vk: int = id r_03;
	r_03: int = add vj v3;
	vb: int = id r_02;
	r_02: int = id vf;
	r_02: int = id r_02;
	ve: int = id r_01;
	vc: int = id r_01;
	r_01: int = id ve;
	r_01: int = add r_01 v0;
	vd: int = id r_01;
	r_01: int = id vc;
	r_01: int = id r_01;
	vm: int = id r_03;
	vk: int = id r_03;
	r_03: int = id vm;
	r_03: int = add r_03 v0;
	vj: int = id vb;
	print vj;
	print r_03;
	print r_01;
}
