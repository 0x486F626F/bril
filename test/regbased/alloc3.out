-6
9
3
13
14
